app/src/main/res/layout/content_main.xml

```diff
@@ -8,7 +8,18 @@
     tools:context=".MainActivity"
     tools:showIn="@layout/activity_main">
 
+    <Button
+        android:id="@+id/button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="New button"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/8d3333d0ff432d0557742aed5ff0615effcedf8c/app/src/main/res/layout/content_main.xml#L15"></span>Hardcoded string “New button”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>